types/express-boom/index.d.ts

```diff
@@ -0,0 +1,353 @@
+// Type definitions for express-boom 3.0
+// Project: https://github.com/scottcorgan/express-boom
+// Definitions by: Aurelien Gaston <https://github.com/Nox-404>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/**
+ * Type definition for the middleware itself
+ */
+
+import express = require('express');
+declare namespace e { }
+declare function e(): express.RequestHandler;
+export = e;
+
+/**
+ * Type augmentation for express
+ */
+
+/**
+ * Based on Sandeep K Nair gist: https://gist.github.com/sandeepsuvit/2486c99c0346db87de24539472f34451
+ * And https://github.com/hapijs/boom
+ */
+declare global {
+  namespace Express {
+    interface Boom {
```

Comment:
I feel like I'm missing something obvious, but how are these types intended to be used?  Perhaps some example code could be added as a test?

Reply:
These types augment express typings. When using express boom middleware, res gets augmented with the boom property.
There is an example already.

Reply:
It looks like the augmentation is adding the `boom` property `Express.Response` - is that correct?  The other types just describe the shape of the new property?

It might be nice to test more than one member (i.e. `forbidden`) of the augmentation.

Reply:
@amcasey That's exactly what you said. I added a few other examples.

Reply:
Thanks!